src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetTransferFinancialBreakdownRequest.cs

```diff
@@ -11,6 +11,6 @@ public GetTransferFinancialBreakdownRequest(long accountId)
         {
             AccountId = accountId;
         }
-        public string GetUrl => $"accounts/{AccountId}/accountprojection/detail?numberOfMonths=12&startDate={DateTime.UtcNow.ToFinancialYear()}-04-06";
+        public string GetUrl => $"accounts/{AccountId}/accountprojection/detail?numberOfMonths=12&startDate={DateTime.UtcNow.StartOfFinancialYear().Year}-04-06";
```

Comment:
If the extension method always returns 6th April, what is the value of only taking the year here only to append 6th April to it again?

Reply:
Good suggestion. I have changed it now and pushed the code.
DateTime.UtcNow.StartOfFinancialYear().ToString("yyyy-MM-dd")